pkg/license/license.go

```diff
@@ -176,6 +176,22 @@ func FindLicenseFiles(fsys fs.FS) ([]LicenseFile, error) {
 // Returns true/fals if the file is a license file, and the weight value
 // associated with the match, as some matches are potentially more relevant.
 func IsLicenseFile(filename string) (bool, float64) {
+	// Ignore files in these paths
+	ignoredPaths := []string{
+		".virtualenv",
+		"env",
+		"node_modules",
+		"venv",
+	}
+	for _, i := range ignoredPaths {
+		if strings.Contains(filepath.Dir(filename), i) {
```

Comment:
One thing I think we should consider here: right now this is done with `strings.Contains()` on a string of the whole directory path chain. The problem is that this will match even if the `ignoredPaths` elements are part of a different word, like "**env**ious" etc. I think we'd need to do a == match on each path element instead. Maybe a filepath.SplitList and then trying to == match each of the split path elements?

Reply:
or we could do a contains of `/foo/` that should isolate the directory name only :+1:

Reply:
Actually... yes! Originally I was worried that there might be cases where this would be the first or last directory in the path string... but I think it might be good? Since `Dir()` does normalize the path. Does it make absolute? Anyway, something surely that we could check!

Reply:
Done now :+1: